src/platform/user/profile/vet360/components/AddressField/AddressForm.jsx

```diff
@@ -18,6 +19,20 @@ import {
  * Validation is managed in the vet360 reducer
  */
 class AddressForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
This is fine but we don't need to use the constructor to set local state. We can just set state as a class property
```
state = {
    isMilitaryAddress: false,
    addressFromRedux: {
      stateCode: this.props.address.stateCode,
      city: this.props.address.city,
      countryName: this.props.address.countryName,
      zipCode: this.props.address.zipCode,
    },
  };
```
more info here: https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599